simplex-messaging.md

```diff
@@ -118,17 +115,6 @@ This protocol also can be used for off-the-record messaging, as Alice and Bob ca
 How simplex connections are used by the participants (graph vertices) is defined by graph-chat protocol and is not in scope of this low level simplex messaging protocol.
 
 
-## Alternative flow to establish a simplex connection
```

Comment:
I'm still not fully convinced it's such a bad idea to have this around even if it's a premature optimization and we don't want to spend time implementing it yet. Maybe mark the whole section as `[out of scope for protocol v1 implementation]` or smth along those lines?

Reply:
I don't think we should until we see that the performance of the standard flow is not good enough (and I believe it will be fast anyway). Any extra flow to maintain is an additional security risk to manage, test, debug, analyse, etc. - better not have it.

Reply:
okay, well, right.. anyway it'd be possible to restore it via vcs..